File: Code/SimpleBudget.API/Models/ProductPrices/ProductPriceGridModel.cs
namespace SimpleBudget.API.Models
{
    public class ProductPriceGridModel
    {
        public int ProductPriceId { get; set; }
        public string ProductName { get; set; } = default!;
        public string? CompanyName { get; set; }
        public string? CategoryName { get; set; }
        public decimal Price { get; set; }
        public string PriceDate { get; set; } = default!;
        public bool IsDiscount { get; set; }
        public int? Quantity { get; set; }
        public string? Description { get; set; }
    }
}

File: Code/client/src/api/models/ProductPriceGridModel.ts
export interface ProductPriceGridModel {
    productPriceId: number;
    productName: string;
    companyName: string | undefined | null;
    categoryName: string | undefined | null;
    price: number;
    priceDate: string;
    isDiscount: boolean;
    quantity: number | undefined | null;
    description: string | undefined | null;
}